app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -320,7 +322,13 @@ class CtaViewModel @Inject constructor(
             }
 
             if (areInContextDaxDialogsCompleted()) {
-                return if (brokenSitePrompt.shouldShowBrokenSitePrompt(nonNullSite.url)) {
+                val detectedRefreshPatterns = brokenSitePrompt.getUserRefreshesCount()
```

Comment:
I don't think we should do this here. Otherwise, we lose signal on breakage for users that haven't completed onboarding